lms/djangoapps/experiments/utils.py

```diff
@@ -378,21 +434,9 @@ def get_experiment_user_metadata_context(course, user):
     }
 
 
-def get_audit_access_expiration(user, course):
```

Comment:
Why did you move this function?

Reply:
To get the base "create context" methods next to each other in the file. Not a big deal either way.